sql/src/main/java/io/crate/execution/ddl/tables/AlterTableOperation.java

```diff
@@ -463,19 +463,21 @@ private static DiscoveryNode getNodeForResize(ClusterState state) {
                         ActiveShardCount.ONE,
                         request.ackTimeout(),
                         shardsAcked -> {
-                            if (!shardsAcked && logger.isInfoEnabled()) {
-                                logger.info("[{}] indexes shrinked, but the operation timed out while waiting for " +
+                            if (!shardsAcked) {
+                                logger.warn("[{}] indexes shrinked, but the operation timed out while waiting for " +
                                             "enough shards to be started. Timeout={}. " +
-                                            "Consider increasing the timeout or adding nodes to the cluster.",
+                                            "Consider increasing the timeout or adding nodes to the cluster and retry the operation",
                                     request.getTargetIndexRequest().indices(), request.timeout());
+                                throw new RuntimeException("Shrinking Request failed");
```

Comment:
I think we should be more specific here - The shink worked but timed out whle waiting for active shards. We can tell that to the user. And since we propagate that information to the user we don't have to log it.